src/charts/tooltip/Tooltip.js

```diff
@@ -117,6 +117,12 @@ export default class Tooltip extends React.Component {
                 customMouseOver: this._handleMouseOver.bind(this),
             });
         }
+
+        this._rootNode = null;
+
+        this._setRef = componentNode => {
```

Comment:
I would prefer to keep this as a function, but calling it here as you are doing.

```
constructor(props) {
   ...
   this._setRef();
}
_setRef = (componentNode) => {
...
}
```

Reply:
+1

Doing as @Golodhros suggested should be fine since it will transpile down to essentially the same thing: https://medium.com/@jacobworrel/babels-transform-class-properties-plugin-how-it-works-and-what-it-means-for-your-react-apps-6983539ffc22

Reply:
Hi @Golodhros thanks for taking the time to review the PR.

I tested the way you suggest but the problem is that calling the function without the parameter will result in an error for not being able to declare `_rootNode` from `undefined`.

A solution we can apply to maintain the `_setRef` as a class property is to explicitly bind the function in the constructor, this way we don't even have to declare the `_rootNode` beforehand with null.

```javascript
constructor(props) {
   ...
   this._setRef = this._setRef.bind(this);
}
...
_setRef = (componentNode) => {
	...
}

```

Let me know if you are ok with this approach.

Reply:
Hi @davegomez,

Thanks for the follow up. I guess if we bind it in the constructor we won't need to use the arrow function when we define the class function.

We don't have the set up to test it, so whatever you that works and is aligned to the best practices will be good for us.

Thanks!

Reply:
@Golodhros the binding in the constructor proposal is with a normal function declaration in order to maintain the same code style.

I ran the tests and the lint and everything passed.

The commit was amend so you can merge if you like the fix. 😊